cdc/sorter/leveldb/system/system.go

```diff
@@ -227,6 +235,10 @@ func (s *System) Stop() error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+	err = s.ReaderSystem.Stop()
```

Comment:
shall we ignore the error and stop all actor systems

Reply:
It may leaks some resouces, like iterators.